### ISSUE TITLE
thrown? in clojure.test/are is reported as unresolved symbol

### ISSUE THREAD

**Issue by liquidz** [borkdude/clj-kondo#626]
**version**

v2019.11.23

**platform**

Ubuntu 18.04

**editor**

vim

**problem**

`thrown?` in `clojure.test/are` is reported as an unresolved symbol in v2019.11.23, while it is not reported in v2019.11.07.

**repro**

* core_test.clj
```clojure
(ns foo.core-test
  (:require [clojure.test :as t]))

(t/deftest a-test
  (t/is (thrown? Exception :foo))
  (t/are [x] (thrown? Exception x) :foo))
```

* Run clj-kondo without any configuration files.
```terminal
$ clj-kondo --lint core_test.clj
core_test.clj:6:15: error: unresolved symbol thrown?
linting took 13ms, errors: 1, warnings: 0
```

**expected behavior**

```terminal
$ clj-kondo-v2019.11.07 --lint core_test.clj 
linting took 13ms, errors: 0, warnings: 0
```
